packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -68,78 +76,97 @@
                     color-scheme="positive" />
             </div>
 
+            <template v-if="isLoading">
+                <restaurant-content-skeleton
+                    :width-percentage="90"
+                    :class="[
+                        $style['c-restaurantCard-content--left-col'],
+                        $style['c-restaurantCard--hideOnMob']]" />
+                <restaurant-content-skeleton
+                    :width-percentage="80"
+                    :class="[
+                        $style['c-restaurantCard-content--left-col'],
+                        $style['c-restaurantCard--hideOnMob']]" />
+                <restaurant-content-skeleton
+                    :width-percentage="40"
+                    :class="[
+                        $style['c-restaurantCard--hideOnMob']]" />
+                <restaurant-content-skeleton :width-percentage="90" />
+            </template>
```

Comment:
Vue Fragments would be really useful here, because it might be nice to have this inside of a component but that would require a parent div which could mess up the styling 😄

Reply:
yeah it would indeed mess up the layout, and I wanted to preserve all the grid/flex styling already in place :)